src/renderer/src/routes/_Map.test.tsx

```diff
@@ -0,0 +1,93 @@
+import type { ReactNode } from 'react'
+import { describe, expect, test, vi } from 'vitest'
+
+import { App, router } from '../App'
+import {
+	PersistedProjectIdContext,
+	nonPersistedProjectIdStore,
+} from '../contexts/persistedState/PersistedProjectId'
+import { render, screen } from '../test/test-util'
+
+// import { useDeviceInfo } from '../queries/deviceInfo'
+
+nonPersistedProjectIdStore.setState(() => ({
+	projectId: 'tester',
+}))
+
+vi.mock('../queries/deviceInfo', () => ({
+	useDeviceInfo: vi.fn(() => ({ data: { name: 'erik' } })),
+}))
+
+const Wrapper = ({ children }: { children: ReactNode }) => (
+	<PersistedProjectIdContext.Provider value={nonPersistedProjectIdStore}>
+		{children}
+	</PersistedProjectIdContext.Provider>
```

Comment:
this simulates PersistedProjectIdStore by using the same context that propagates the persisted store, but injecting a non persisted store for testing purposes